src/mod/applications/mod_conference/conference_file.c

```diff
@@ -149,6 +149,8 @@ uint32_t conference_file_stop(conference_obj_t *conference, file_stop_t stop)
 	return count;
 }
 
+static switch_bool_t do_print = SWITCH_TRUE;
```

Comment:
How should this global var work? Per thread/per conference or somehow else?

Reply:
Made declaration explicitly per conference.